base/sparse/cholmod.jl

```diff
@@ -418,6 +418,13 @@ function copy_dense{Tv<:VTypes}(A::Dense{Tv})
     d
 end
 
+function sort!{Tv<:VTypes}(S::Sparse{Tv})
```

Comment:
use a more descriptive name for this? confusing to locally shadow generic functions but with different meanings